low-level/files/opt/seagate/sspl/bin/conf_based_sensors_enable

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e -u -o pipefail
+
+to_lower_case() {
+    tr '[:upper:]' '[:lower:]'
+}
+
+declare -a sed_expr=(-e '/^\[[A-Z]\+\]$/h')
+storage_type="$(provisioner get_setup_info --output json | jq -r .ret.storage_type | to_lower_case)"
+server_type="$(provisioner get_setup_info --output json | jq -r .ret.server_type | to_lower_case)"
+
+declare -A monitor=(
+    [REALSTORSENSORS]="true"
+    [NODEHWSENSOR]="true"
+    [SYSTEMDWATCHDOG]="true" # This is the 'server disks' sensor
+    [RAIDSENSOR]="true"
+    [SASPORTSENSOR]="true"
+    [MEMFAULTSENSOR]="true"
+    [CPUFAULTSENSOR]="true"
+
+)
+
+case "${storage_type}" in
+    virtual) 
+        monitor[REALSTORSENSORS]="false"
+        monitor[SYSTEMDWATCHDOG]="false"
+        ;;
+    jbod)
+        monitor[REALSTORSENSORS]="false"
+        monitor[SYSTEMDWATCHDOG]="false"
+        monitor[RAIDSENSOR]="false"
```

Comment:
SYSTEMDWATCHDOG & RAID sensor are for server disks & RAID arrays made of server disks again, why being disabled for external JBOD storage type ?

Reply:
i can recall this, so guess we are fine

> During JBOD discussions, which would be the case for most open src users, call was taken to keep minimal set of sensors enabled. Wherever open src users would install Cortx, may not have RAIDed disks, or have very less possibility and so the sensor we would disable by default.

Reply:
SystemdWatchdog should be enabled for JBOD.
RAID should be disabled for 'server_type' == 'virtual'. It should not depend on 'storage_type' at all.
As discussed.